Hashing-2-problem-3.txt

```diff
@@ -0,0 +1,34 @@
+
+## Problem3 (https://leetcode.com/problems/longest-palindrome)
+
+##TC:O(N) where N is the length of the String
+
+##SC:O(N) where N is the length of the string
```

Comment:
Space will be O(1). You are storing characters and at max, you can have 26 characters.